travis-ci/install.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-BAZEL_URL=https://github.com/bazelbuild/bazel/releases/download/0.3.0/bazel-0.3.0-installer-linux-x86_64.sh
+BAZEL_URL=https://github.com/bazelbuild/bazel/releases/download/0.3.2/bazel-0.3.2-installer-linux-x86_64.sh
```

Comment:
Use the 0.4.3 release? I've personally run into strange bugs on the 0.3.x release.